src/helics/core/CommonCore.cpp

```diff
@@ -1992,7 +1998,8 @@ void CommonCore::loadBasicJsonInfo(
 
 void CommonCore::initializeMapBuilder(const std::string& request,
                                       std::uint16_t index,
-                                      bool reset) const
+                                      bool reset,
+                                      bool synchronous) const
```

Comment:
I am going to rename the synchronous to something else that makes a little more sense now